### PR TITLE
feat: multiple images in offer

### PR DESCRIPTION
**Pull request by vladimir-buravlev**

### CODE REVIEW

File: apps/vendor-portal/src/modules/offers/pages/offers-details.vue [hunk 1]
```diff
@@ -675,17 +672,55 @@ async function onImageUpload(files: FileList) {
...
+    component: AssetsDetails,
+    componentOptions: {
+      editableAsset: item,
+      images: offerDetails.images,
+      sortHandler: sortImage,
+    },
+  });
+};
+
+function sortImage(remove = false, localImage: IImage) {
```
Comment (tatarincev) [inline apps/vendor-portal/src/modules/offers/pages/offers-details.vue:688]: I think this logic must be moved out of here into composable level.

File: apps/vendor-portal/src/modules/offers/pages/offers-details.vue [hunk 2]
```diff
@@ -675,17 +672,55 @@ async function onImageUpload(files: FileList) {
...
+};
+
+const onGalleryImageRemove = (image: Image) => {
   if (
     window.confirm(
       unref(computed(() => t("OFFERS.PAGES.ALERTS.IMAGE_DELETE_CONFIRMATION")))
     )
   ) {
-    offerDetails.imgSrc = undefined;
+    const imageIndex = offerDetails.images.findIndex((img) => {
```
Comment (tatarincev) [inline apps/vendor-portal/src/modules/offers/pages/offers-details.vue:714]: I think this logic must be moved out of here into composable level.

**Review by tatarincev — CHANGES_REQUESTED** [review]

File: apps/vendor-portal/src/modules/offers/pages/offers-details.vue (hunk 1 above)
Comment (vladimir-buravlev) [reply apps/vendor-portal/src/modules/offers/pages/offers-details.vue]: done

File: apps/vendor-portal/src/modules/offers/pages/offers-details.vue (hunk 2 above)
Comment (vladimir-buravlev) [reply apps/vendor-portal/src/modules/offers/pages/offers-details.vue]: done

**Review by tatarincev — APPROVED** [review]

**Pull request merged**